lib/lotus/routing/resource/action.rb

```diff
@@ -179,7 +179,7 @@ def rest_path
         # @api private
         # @since 0.1.0
         def as
-          namespace.relative_join(resource_name, self.class.named_route_separator).to_sym
+          namespace.relative_join(resource_name.to_s.gsub("/","_"), self.class.named_route_separator).to_sym
```

Comment:
@AlfonsoUceda Can we please give a name to `/` and `_` and extract in Ruby constants? I think that we'll easy forget about the meaning of those chars. Using frozen strings as constants helps with perf. Thank you! :+1: